client/apps/edit/public/icons/edit_section_remove.coffee

```diff
@@ -0,0 +1,25 @@
+React = require 'react'
+{ svg, circle, polygon } = React.DOM
+
+module.exports = React.createClass
+	displayName: 'EditSectionRemoveIcon'
+
+	render: ->
+  	svg {
```

Comment:
While I find it pleasing to the 👀  to see svg in coffeescript—maybe we could leave these icons as .svg files and load them with fs.readFileSync via [brfs](https://github.com/substack/brfs).